hr_time/public/scss/hr_time.bundle.scss

```
@@ -1,2 +1,32 @@
+@import 'mixins';
 @import "./dashboard";
-@import "./checkin_status";
\ No newline at end of file
+@import "./checkin_status";
+
+.modal .modal-body{
+    padding-bottom: 0 !important;
+}
+
+label#worklog_section_label{
+    margin-bottom: 0 !important;
+}
+
+.worklog-status-alert{
+    @include flex-center;
+    @include badge;
+    opacity: 0.9;
+    font-size: 0.7rem;
+    margin-top: 1rem;
+    margin-bottom: 0;
+    border: 1px solid #dee2e6;  /* border with default color */
```

We should try to use default colors. Seems just using the CSS clases ist not fully sufficient, so we need to manually adjust it.

```suggestion
    border: 1px solid var(--error-border);
    color: var(--alert-text-danger);
    background: var(--alert-bg-danger);
```